### ISSUE TITLE
Feature req: Toggle game image

### ISSUE THREAD

**Issue by pth0rn** [Herdinger/EmulationStation#33]
Seeing in-game images along with boxart for each game would be great. But since there's not much screen real estate as is, there could be a system wide toggle to switch between showing box-art and an in-game image

**Comment by kriptonian** [comment]
Hello, I wrote weeks ago the code to manage a game screenshot (not the toggle, unfortunately).
The result is in the screenshot attached.
![schermata1](https://cloud.githubusercontent.com/assets/19637542/15635361/24ac7e66-25dd-11e6-8dbe-1c34b8ef5975.png)

I will be happy to share the code with you all, if you want, but consider that:
1) i'm not a c++ programmer, so someone of you should look at the code and, probably, do a better work tha the one I did
2) I could share the code and the modification to only three themes: simple, carbon, clean canela, but only for the system I'm using: atari 2600, atari 5200, atari 7800, amiga, c64, vic20, mame, snes, megadrive, zx spectrum, ti ti-99/4a. The others must be done. Also the files with the gamelist must be modified adding a new tag "screenshot" for each game, if you want to view them: without this tag, the space reserved to the screenshot will be empty.
3) At the moment there no switch to enable/disable the screenshot: the description text has the half in width so programmatically it could be doubled in width while disabling the screenshot
4) there no code to manage the editing of the screenshot inside the game information card and there no code about scraping. I used Linux bash to copy the "box art" tag  from the gamelist xmls and modify the "screenshot" tag with some text substitutions. The boxart file and the screenshot file have the same name and the same file type (different formats were converted by bash command mogrify), so it was simple to manage without need of writing that implementation. I could share this bash commands, if I did not delete the scripts, I don't remember at this moment. This code inside emulationstation must be created.
5) there is no manage of the occupation in case of absence of the screenshot: the space reserved will be empty. Without it, perhaps, it should be better that the description could fullfill the entire area: it's not my case, all games have their screenshot.
6) I never used GitHub (it's my first post), so I don't know how sharing everything :-(

To summarize, all the situations not needed by my system were not implemented, I had no time to do it, unfortunately. So the best approach to solve these sceneries must by decided, and then they must be implemented, for a better usability, by someone skilled.

Let me know if you want the code I wrote.

**Comment by blm07** [comment]
What if we had an option to show a screen shot/title screen rather than the text description?
